_config.yml

```diff
@@ -0,0 +1,29 @@
+title: Flyte by Lyft
+description: A managed platform for executing distributed workflows at scale.
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "https://kylewaynebenson.github.io/flyte" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
we should put the correct url here: @georgesnelling

Reply:
I just pushed two commits: one to correct the URL, one to simplify deployment.